releasenotes/notes/deprecate-stabilizer-table-9efd08c7de1a5b4d.yaml

```diff
@@ -0,0 +1,9 @@
+---
+deprecations:
+  - |
+    The :class:`~qiskit.quantum_info.StabilizerTable` is deprecated and will be removed in a future
+    release. Instead, the :class:`~qiskit.quantum_info.PauliList` should be used.
+    With this change, :attr:`.Clifford.table`, :attr:`.Clifford.stabilizer`, and
+    :attr:`.Clifford.destabilizer` have been deprecated so that you should use
+    :attr:`.Clifford.tableau`, :attr:`.Clifford.stabilizers`, :attr:`.Clifford.destabilizers`.
```

Comment:
The methods now are called `Clifford.stab` and `Clifford.destab`?
Also, there are other new methods added.